core/src/main/java/com/amplifyframework/core/model/query/QuerySortBy.java

```diff
@@ -32,6 +34,7 @@
  * {@link QueryField#descending()} helper methods (e.g. Todo.DESCRIPTION.ascending() or Todo.DESCRIPTION.descending())
  */
 public final class QuerySortBy {
+    private final String modelName;
```

Comment:
Rather than add a `String modelName` property, what do you think about replacing the `String field` property with a `QueryField queryField` property (which contains the `modelName`, and `fieldName`)?   

Maybe slightly cleaner, though not a huge deal either way as this is just an internal implementation detail.